package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/accepts-serializer",
-  "version": "5.2.0",
+  "version": "5.3.0",
```

Comment:
Please do not change the version here.
It should be handled by the releaser.

Reply:
Do you want me to submit a new PR or can we change it here directly?